compose-Satosa-Saml2Spid/nginx/certs/proxy_local.key

```diff
@@ -0,0 +1,40 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Io rimuoverei queste e le farei creare a mano o ottenere con letencrypt con una sezione di README chiara

Reply:
Con un certificato già presente hai la possibilità di far partire la composizione subito dopo il pull del progetto. Solo per provarlo e magari per vedere come vanno poi messe le tue configurazioni, senza pensare ad altro. Certo per metterlo definitivo dovremo scrivere bene cosa cambiare a dove. Avevo pensato di inserire un container certbot nella composizione per automatizzare il tutto. Magari commentato cosi chi lo vuole utilizzare lo può fare, ma se magari devi solo "provarlo" in locale puoi lasciare il certificato fasullo già fornito. Comunque si potrebbe unire le due idee. Togliere crt e key, indicare bene nel README (magari in un README nella directory compose-Satosa-Saml2Spid), che prima di lanciarlo occorre creare il crt. E mettere comunque nel docker-compose.yml un container certbot con una configurazione base da customizzare per dare già un prodotto pronto alla produzione?